src/Aspire.Dashboard/Components/Controls/SummaryDetailsView.razor.cs

```diff
@@ -160,6 +179,88 @@ private void SetPanelSizes(float panel1Fraction)
         _panel2Size = string.Create(CultureInfo.InvariantCulture, $"{(1 - panel1Fraction):F3}fr");
     }
 
+    public async Task OnPageKeyDownAsync(KeyboardEventArgsWithPressedKeys args)
+    {
+        if (_splitterRef is null || !args.ShiftKey)
+        {
+            return;
+        }
+
+        if (args.Key.ToLower(CultureInfo.InvariantCulture) == "t")
+        {
+            await HandleToggleOrientation();
+            return;
+        }
+
+        if (args.Key.ToLower(CultureInfo.InvariantCulture) == "o")
```

Comment:
If I follow these steps:

1. Run the app
2. Click Details on any resource
3. Press shift+o

The details panel will close (as expected) and then immediately re-open. It seems like this is reliably reproducible.

Reply:
Thanks for catching this. FIxed